sqlalchemy_jsonapi/unittests/test_serializer_post_collection.py

```diff
@@ -40,5 +83,208 @@ def test_201_resource_creation(self):
         }
 
         actual = response.data
-        self.assertEqual(expected_response, actual)
-        self.assertEqual(201, response.status_code)
+        try:
```

Comment:
Another api suggestion would be to express this as a decorator, something like `@fragile` that works as annotation as well.